client/gdc/bam.js

```diff
@@ -780,12 +786,16 @@ export async function bamsliceui({
 		for (const [gene, mlst] of gene2mlst) {
 			for (const m of mlst) {
 				const row = []
+				// address section 508 requirement to have explicit or implicit labels for input elements
+				const elemId = `${gene}-${m.mname}`.replace(/\W+/g, '_')
+				row.ariaLabelledBy = elemId
 				row.push({ value: gene, data: m })
-				row.push({ value: m.mname })
+				row.push({ value: m.mname, elemId })
 				row.push({ value: mclass[m.class]?.label || 'Unknown' })
 				row.push({ value: m.chr + ':' + m.pos + ' ' + m.ref + '>' + m.alt })
 				rows.push(row)
 			}
+			if (rows.length > 5) break
```

Comment:
this truncates mutation table. why?

Reply:
sorry, will fix, I truncated only for testing

Reply:
fixed